apps/storybook-react/tailwind.config.js

```diff
@@ -10,7 +10,15 @@ module.exports = {
     './stories/**/*.{js,jsx,ts,tsx}',
   ],
   theme: {
-    colors: {}, // This removes all default Tailwind colors. We want to rely on the colors provided by @metamask-previews/design-system-tailwind-preset
+    // Keep essential semantic colors, remove default palette colors. We want to rely on the colors provided by @metamask-previews/design-system-tailwind-preset
+    colors: {
+      inherit: 'inherit',
+      current: 'currentColor',
+      transparent: 'transparent',
+      black: '#000000',
+      white: '#ffffff',
+    },
+    fontSize: {}, // This removes all default Tailwind font sizes. We want to rely on the design system font sizes and enforce use of the Text component
```

Comment:
We remove all default font sizes to enforce the use of the Text component. We aren removing the below there are no `inherit` or other semantic default tailwind classnames for [font size](https://v3.tailwindcss.com/docs/font-size#class-reference)
```
text-xs	
text-sm	
text-base	
text-lg	
text-xl	
text-2xl	
text-3xl	
text-4xl	
text-5xl	
text-6xl	
text-7xl	
text-8xl	
text-9xl
```